packages/typespec-ts/test/nonBrandedIntegration/modular/generated/models/usage/src/rest/usageClient.ts

```diff
@@ -10,7 +10,8 @@ import { UsageContext } from "./clientDefinitions.js";
 export default function createClient(
   options: ClientOptions = {},
 ): UsageContext {
-  const baseUrl = options.baseUrl ?? `http://localhost:3000`;
+  const endpointUrl =
+    options.baseUrl ?? options.baseUrl ?? `http://localhost:3000`;
```

Comment:
endpoint doesn't exist in ClientOptions
![image](https://github.com/Azure/autorest.typescript/assets/98569699/1002a581-5bdb-48cd-8360-072f23495568)

Reply:
we need to update the non-brand core